pylib/mozhginfo/mozhginfo/push.py

```diff
@@ -0,0 +1,62 @@
+class Push(object):
```

Comment:
Can you also create a Changeset class?
With properties for each of these:
        {
          "author": "Mason Chang <mchang@mozilla.com>",
          "branch": "default",
          "desc": "Bug 1127151. Part 1: Expose dwmFlush to winutils. r=jrmuizel",
          "files": [
            "widget/windows/WinUtils.cpp",
            "widget/windows/WinUtils.h"
          ],
          "node": "c4061f7c942252920028d6f73f54902a242b5256",
          "tags": []
        },